Polys: Improved the docstrings of LM(), LT() and revert()(#11427)

shrutishrm512 opened the pull request:
Added more description about the method in LM(), LT() and gave the examples for revert(f, n) to clear about its functionality in docstrings. These changes have been made in the polytools.py file in polys.

asmeurer commented:
There are test failures. See the Travis tests. Be sure to configure your text editor to use four spaces for tabs.

asmeurer commented:
And you can run

```
./bin/doctest
```

locally before committing to make sure everything works.

shrutishrm512 commented:
Please can you tell me in detail about Travis tests for making changes in my code.

Shekharrajak commented on sympy/polys/polytools.py:
```diff
@@ -1979,6 +1978,9 @@ def coeff(f, x, n=1, right=False):
     def LM(f, order=None):
         """
         Returns the leading monomial of ``f``.
+	
```
inconsistent use of tabs and spaces in indentation. Use [PEP](https://packagecontrol.io/packages/Python%20PEP8%20Autoformat) plugin to cross check python code format.

shrutishrm512 commented:
@asmeurer  I have again sent the pull request and it has cleared all the checks, then it can be merged into the main code?

shrutishrm512 committed:
Polys: Improved the docstrings of LM(), LT() and revert()(#11427)

Added more description about the method in LM(), LT() and gave the examples for revert(f, n) to clear about its functionality in docstrings. These changes have been made in the polytools.py file in polys.

shrutishrm512 commented:
#11663

shrutishrm512 committed:
Polys: Improved the docstrings of LM(), LT() and revert()(#11427)

Added more description about the method in LM(), LT() and gave the examples for revert(f, n) to clear about its functionality in docstrings. These changes have been made in the polytools.py file in polys.

asmeurer commented:
Thanks. And by the way, that's the same pull request. And there's no need to open a new pull request. Just push new changes to the same branch, and it will update the pull request (that's what has happened here).

shrutishrm512 commented:
OK sir, I will take care of this in furture.


 Sent with Mailtrack
<https://mailtrack.io/install?source=signature&lang=en&referral=shrutishrm512@gmail.com&idSignature=22>

*Shruti Sharma*

*Pre-final Year Student*
*B.Tech.:Computer Science and Engineering*
*The LNM Institute of Information Technology*
Email ID*:shrutishrm512@gmail.com <shrutishrm512@gmail.com>*

On Fri, Oct 7, 2016 at 1:04 AM, Aaron Meurer <notifications@github.com>
wrote:

> Thanks. And by the way, that's the same pull request. And there's no need
> to open a new pull request. Just push new changes to the same branch, and
> it will update the pull request (that's what has happened here).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sympy/sympy/pull/11663#issuecomment-252065699>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AOpkdB-vDjWzBzh-fH4h4tAdj6wiMXdvks5qxU1bgaJpZM4KGoT4>
> .
>
